spec/codecs/cef_spec.rb

```diff
@@ -452,7 +452,7 @@ def validate(e)
       end
     end
 
-    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
+    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 ad.Authentification=MICROSOFT_AUTHENTICATION_PACKAGE_V1_0 ad.Error_,Code=3221225578 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
```

Comment:
@breml if the ArcSight connector is configured to process using complete mode, it will render additional fields notably ad.fieldnames and it not wholly documented as you might have noticed. The documentation that I've referenced earlier does vaguely note that the additional fields carry no significant use (page 14) unless you're using the ArcSight logger to which at most, one may use it for exporting and a meek search. In any case, I might have been a little overprotective to assume that the codec should work for < v2.4 elasticsearch instances hence, the safeguard to remove the dot notations.

Given that we're gearing towards v5 versions of the elastic stack, I guess we could keep the irregular undocumented fields and as you've mentioned, filtered out via filter plugins if necessary. 

Will make the changes and resubmit my PR. Thanks for the feedback. ~N